src/components/TeamPics/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { FC } from "react";
+
+import TRUMAN from "../../assets/images/truman.png";
+import LORENZO from "../../assets/images/lorenzo.png";
+import MARY from "../../assets/images/mary.jpg";
+import SHIV from "../../assets/images/shivanshu.jpg";
+import ATHENA from "../../assets/images/athena.jpg";
+import ADRIAN from "../../assets/images/adrian.jpg";
+import BUSHER from "../../assets/images/busher.jpg";
+import TUZETTE from "../../assets/images/tuzette.jpg";
+import MANJOT from "../../assets/images/manjot.jpg";
+import INTISER from "../../assets/images/Intiser.png";
+import ALISSON from "../../assets/images/alisson.png";
+import MARTHEN from "../../assets/images/marty.jpg";
+import LAURENCE from "../../assets/images/laurence.png";
+import JET from "../../assets/images/jet.jpg";
+import JOSEPH from "../../assets/images/joseph.jpg";
+import JANELLE from "../../assets/images/janelle.jpg";
+
+import "./styles.scss";
+
+const TeamPics: FC = (): JSX.Element => {
+    const team: Array<{ url: string; src: string }> = [
```

Comment:
Supernit: You should probably declare this outside the component, as it causes an additional memory allocation event even though the data is not changing

Reply:
did not know tyty